docs/filter_kgraph_percentile.md

```diff
@@ -0,0 +1,63 @@
+# filter kgraph (percentile)
+
+_subclass of [filter_kgraph](./filter_kgraph.md)_
+
+This operation removes kgraph edges that have attribute values are below/above the given percentile.
+
+### examples
+
+- [input](../examples/fill_and_filter/messages/05_filtered_kgraph_stat_input.json), [output](../examples/fill_and_filter/messages/06_filtered_kgraph_std_dev_output.json)
+
+### input requirements
+
+None
+
+### output guarantees
+
+- no edges in the kgraph which have attribute values are below/above the given percentile.
+
+### allowed changes
+
+- remove knodes
+- remove kedges
+
+### parameters
+
+```yaml
+properties:
+  edge_attribute:
+    description: The name of the edge attribute to filter on.
+    example: normalized_google_distance
+    type: string
+  edge_keys:
+    default: null
+    description: This indicates if you only want to filter on specific edge_keys.
+      If not provided or empty all edges will be filtered on and edge_key will not
+      be considered when filtering.
+    example:
+    - e01
+    type: array
+  qnode_keys:
+    default: null
+    description: This indicates if you only want nodes corresponding to a specific
+      list of qnode_keys to be removed. If not provided oe empty no nodes will be
+      removed when filtering.
```

Comment:
Add the following: Allows you to choose what happens to source and target nodes attached to edges that get filtered out. Only applicable to qnodes that are either sources or targets of edges with the provided `edge_attribute`.